### PR TITLE
Update published contigs

### PR DESCRIPTION
**Pull request by Ge94**
Small PR to be put in production to solve a small bug. Previously, contigs filtered by human were published. We want those filtered by host too.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/ebi-metagenomics/miassembler/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/assembly_qc.nf [hunk 1]
```diff
@@ -84,10 +84,10 @@ workflow ASSEMBLY_QC {
     }
 
     PUBLISH_CLEANED_CONTIGS(
-        filtered_contigs
+        cleaned_contigs
     )
 
     emit:
-    filtered_contigs = filtered_contigs
+    filtered_contigs = cleaned_contigs
```
Comment (mberacochea) [inline subworkflows/local/assembly_qc.nf:91]: We should also rename the output to avoid having to remember this name mapping outside the module

**Review by mberacochea — APPROVED** [review]
LGTM... left one comment about the output naming.

File: subworkflows/local/assembly_qc.nf (hunk 1 above)
Comment (Ge94) [reply subworkflows/local/assembly_qc.nf:91]: To which output are you referring to, the one from this subwf or the one for published contigs? @mberacochea

Comment (mberacochea) [reply subworkflows/local/assembly_qc.nf:91]: This one, in the emit:
```
filtered_contigs = cleaned_contigs
```
should be
```
cleaned_contigs = cleaned_contigs
```

**Pull request merged**